scripts/install.py

```diff
@@ -137,22 +144,29 @@ def main():
         install_gc3pie_from_pypi(options.target, options.features)
 
     print("""
-===============================
-Installation of GC3Pie is done!
-===============================
-
-In order to work with GC3Pie you have to enable the virtual
-environment with the command:
-
-    . {target}/bin/activate
-
-You need to run the above command on every new shell you open before
-using GC3Pie commands, but just once per session.
+            ===============================
```

Comment:
Text withing this string should keep its (non) indentation.  The text is displayed to users when running the script, indentation according to the source level in Python makes no visual sense.